storage/Utils/SystemCmd.cc

```diff
@@ -561,6 +617,33 @@ namespace storage
     }
 
 
+    void
+    SystemCmd::sendStdin()
+    {
+        if ( ! _childStdin )
+            return;
+
+        if ( ! _stdinText.empty() )
+        {
+            string::size_type count = 0;
+            string::size_type len   = _stdinText.size();
+            int result = 1;
+
+            while ( count < len && result > 0 )
+                result = fputc( _stdinText[ count++ ], _childStdin );
```

Comment:
After we talked about this, is this comment still relevant? If not, please delete.